docs/gen-client-docs.py

```diff
@@ -388,7 +388,7 @@ def _get_tag_name(self, class_name, service_name) -> Tag:
             class_name = class_name.replace('Ext', 'API')
         class_name = class_name[:-3]
         for tag_name, t in self.mapping.items():
-            if t.service == class_name:
+            if t.service.lower() == str(class_name).lower():
```

Comment:
Highlight

Reply:
Note that the SDK has tools to manipulate string casing: https://github.com/databricks/databricks-sdk-py/blob/main/databricks/sdk/casing.py. However, I think this is still safe enough as we shouldn't have services that are the same modulo case.